# Índice de Educación para el municipio de León, Guanajuato

## PNUD

El Programa de las Naciones Unidas para el Desarrollo es el organismo mundial de las Naciones Unidas en materia de desarrollo que promueve el cambio y conecta a los países con los conocimientos, la experiencia y los recursos necesarios para ayudar a los pueblos a forjar una vida mejor.
![alt text][logo]

[logo]: https://onu.org.gt/wp-content/uploads/2015/11/PNUD.png

Está presente en unos 170 países y territorios, al apoyar sus propias soluciones ante los desafíos relativos al desarrollo y al crear capacidades a nivel nacional y local, que les ayudarán a lograr el desarrollo humano y los Objetivos de Desarrollo Sostenible. Nuestro trabajo se enfoca en tres áreas principales:

* Desarrollo sostenible.
* Gobernabilidad democrática y consolidación de la paz.
* Resiliencia ante el clima y los desastres.

El PNUD ayuda a los países a obtener y utilizar la asistencia efectivamente. En todas sus actividades, promueven la igualdad de género y la protección de los derechos humanos.

El Índice de Desarrollo Humano (IDH) se creó para hacer hincapié en que la ampliación de las oportunidades de las personas debería ser el criterio más importante para evaluar los resultados en materia de desarrollo. El crecimiento económico es un medio que contribuye a ese proceso, pero no es un objetivo en sí mismo.

El IDH mide el progreso conseguido por un país en tres dimensiones básicas del desarrollo humano: disfrutar de una vida larga y saludable, acceso a educación y nivel de vida digno. Por lo tanto el IDH se conforma de tres indices:

* Índice de Salud
* Índice de Educación
* Índice de Ingreso

## León, Guanajuato

![alt text][leon]

[leon]: https://www.implan.gob.mx/images/Logo_Ppresidencia_2015-2018-01.png

León, oficialmente León de Los Aldama, también conocida como la Perla y Motor del Bajío o la Capital Mundial del Calzado, es una ciudad mexicana, cabecera del municipio homónimo, ubicada en el Estado de Guanajuato. La zona metropolitana de León cuenta con poco más de 2,115,000 habitantes en 2019, solo la ciudad de León cuenta con 1,800,000 habitantes lo que la convierte en la localidad más poblada del estado y la séptima más poblada del país. Se encuentra a 384 km de la capital del país, la Ciudad de México. La ciudad está entre las 5 ciudades más importantes de México, junto con la Ciudad de México, Guadalajara, Monterrey y Puebla, así como también es la ciudad más grande e importante de el Bajío.

Esta localidad forma parte del Área Metropolitana de León que aunque aún no estén juntas del todo las ciudades que forman el área metropolitana de León, la mayor economía se concentra en Leon (Silao de la Victoria, San Francisco del Rincón, Purísima del Rincón), conforma la séptima metrópoli más grande del país y la mayor de la macro región del Bajío, con poco más de 2,115,000 habitantes.

Fue fundada el 20 de enero de 1576 por Martín Enríquez de Almansa bajo el nombre de Villa de León, recibió su nombre actual en 1830 en honor a los insurgentes Juan Aldama e Ignacio Aldama, quienes participaron en la guerra de Independencia de México. Desde la década de los años 40 del siglo XX, la economía de la ciudad ha girado básicamente alrededor de la cadena productiva del cuero, calzado, proveeduría y marroquinería —por eso se la llama La Capital Mundial del Calzado—, aunque para inicios del siglo XXI, comienza a reorientar su economía a los servicios y la industria automotriz.

Este lugar, está consolidando otras vocaciones económicas: la atención de la salud de alta especialidad y diagnóstico, la educación superior y la investigación, así como el turismo de negocios. Pese a que no es la capital política del estado, se le considera la capital económica, social, comercial y de servicios.

### Índice de Desarrollo Humano municipal en León

Se levantó una encuesta por parte de los alumnos de la Licenciatura en Economía en algunas colonias marginadas representativas de la ciudad de León que pretende tener la información para calcular en Índice de Desarrollo Humano de estas colonias.

Esta encuesta busca obtener datos para poder obtener diversos indicadores de desarrollo multidimensionales, entre ellos el Índice de Desarrollo Humano.

**Como parte del Diplomado en Economía Aplicada** utilicé esta encuesta para calcular el Índice de Educación que es parte del Índice de Desarrollo Humano. Este indice se define como:

<br>
<center>$(\frac{2}{3})\frac{Población Alfabeta De 15 años Y Más}{Población De 15 Años Y más}+(\frac{1}{3})\frac{Población De 6 a 24 Años Escolarizada}{Población de 6 a 24 años}$</center>
<br>
Para el cálcuo de este índice se utiliza el software estadístico R. La base de datos contiene los años de educación de los abuelos, los padres y los hijos de las familias encuestadas.
Dado a que la información recabada en la encuesta no es lo suficientemente detallada para calcular el índice de manera literal, tuve que hacer ciertos supuestos en la información para poder obtener una buena aproximación al índice. 

Los supuestos son los siguientes:

* Se considera alfabetizado al Abuelo y Padre con uno o más años de educación
* Se considera mayor de 15 años, aquello hijos que tengan 9 o más años de educación, ya que se asume que la educación comienza a los seis años
* Todos los hijos se consideran que tienen entre 6 y 24 años
* Cuando no se muestra los años de educación con un NA, se considera que siguen asistiendo a la escuela

# Código de R

In [9]:
d<-read.csv("Leon.csv",header=T)
#poblacion mayor a quince años alfabetizada
pob_alf=0
for (i in 1:134){
  if (d[i,102]>=1){
    pob_alf=pob_alf+1
  }
  if (d[i,103]>=1){
    pob_alf=pob_alf+1
  }
  if (!is.na(d[i,104])){
    if (as.numeric(d[i,104])>=9){
      pob_alf=pob_alf+1
    }
  }
}
print("Poblacion Alfabetizada mayor a 15 años: ")
pob_alf
#Poblacion mayor a quince años
pob_15=0
for (i in 1:134){
  if (d[i,102]>=0){
    pob_15=pob_15+1
}
  if (d[i,103]>=0){
    pob_15=pob_15+1
  }
  if (!is.na(d[i,104])){
    if (as.numeric(d[i,104])>=9){
      pob_15=pob_15+1
    }
  }
}
print("Poblacion mayor a 15 años: ")
pob_15

#poblacion que asiste a la escuela
pob_esc=0
for (i in 1:134){
  if (is.na(d[i,104])){
    pob_esc=pob_esc+1
  }
}
print("Poblacion que asiste a la escuela: ")
pob_esc

#población de hijos
pob6_24=134
print("Poblacion de 6 a 24 años: ")
pob6_24

#Indice de alfabetización de Adultos
print("Indice de alfabetización de Adultos:")
IAA=pob_alf/pob_15
IAA

#Indice de asistencia escolar
print("Indice de asistencia escolar: ")
IAE=pob_esc/pob6_24
IAE

#indice de educacion
index=(2/3)*IAA+(1/3)*IAE
print("Indice de Educación: ")
index


[1] "Poblacion Alfabetizada mayor a 15 años: "


[1] 175

[1] "Poblacion mayor a 15 años: "


[1] 296

[1] "Poblacion que asiste a la escuela: "


[1] 64

[1] "Poblacion de 6 a 24 años: "


[1] 134

[1] "Indice de alfabetización de Adultos:"


[1] 0.5912162

[1] "Indice de asistencia escolar: "


[1] 0.4776119

[1] "Indice de Educación: "


[1] 0.5533481

# Conclusión

En 2010, el PNUD publicó el Índice de Desarrollo Humano Municipal de México. El nivel de desarrollo humano del Estado de Guanajuato se calcula mediante los logros de la entidad, en salud (0.852),
educación (0.611) e ingreso (0.693), alcanzados en relación con los parámetros observados a nivel internacional.
El resultado global para el estado es un Índice de Desarrollo Humano (idh) de 0.712 en 2010, que es comparable
con el nivel de desarrollo de países como Colombia y Túnez.

En educación índice promedio estatal fue de **0.611** , los municipios con mayor índice de educación fueron:

* Guanajuato **(0.727)**
* Celaya **(0.704)**
* Irapuato **(0.672)**

y los de menor índice fueron:

* Maneul Doblado **(0.458)**
* Ocampo **(0.457)**
* Xichú **(0.435)**

De esta manera si comparamos las zonas marginadas a las que les fueron levantadas las encuestas el índice de educación resultó en **0.553**, por lo que se encuentran ligeramente por encima de los municipios con los menores índices de educación, pero por debajo del promedio estatal y muy por debajo de Guanajuato que es el municipio con mayor índice de educación y de desarrollo humano. 
